tox.ini

```diff
@@ -75,8 +75,7 @@ commands =
     python run_pylint.py
 deps =
     pep8
-    -ehg+https://bitbucket.org/logilab/pylint@33e334be064c#egg=pylint
-    logilab-common<=0.63.0  # Hack for pylint upstream
+    pylint
     unittest2
```

Comment:
Thanks @PCManticore! Welcome to GitHub (I didn't realize you had an account here too.)

Our builds weren't broken once I switching to just normal `pylint` (it grabs the freshest version of all deps from PyPI).